float32[] ik_wbc_out
